### PR TITLE
Close #14948

### LINKED ISSUE #14948: put

### PR DESCRIPTION
**Pull request by utkarsh-shrivastav77**
Added the put numpy function

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by utkarsh-shrivastav77** [comment]
Sir please tell me can it be merged

**Comment by utkarsh-shrivastav77** [comment]
I have implemented the test function

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_put_operations.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import ivy_tests.test_ivy.helpers as helpers
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+from ivy.functional.frontends.numpy.func_wrapper import (
+    to_ivy_arrays_and_back,
+)
+import ivy.functional.frontends.numpy as np_frontend
+
+
+@to_ivy_arrays_and_back
```
Comment (tomatillos) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_put_operations.py:12]: You don't need to redefine the function in the test

**Comment by utkarsh-shrivastav77** [comment]
Sir, Please reveiw my code

**Comment by tomatillos** [comment]
Hi, the tests do not pass on the CI, have you got the testing setup working locally?

**Comment by utkarsh-shrivastav77** [comment]
No sir

**Comment by tomatillos** [comment]
OK, please follow the relevant section of: https://unify.ai/docs/ivy/overview/contributing/setting_up.html depending on what setup you have. Once you have the test setup working, you can fix your function, and then hopefully we can merge this!

**Comment by utkarsh-shrivastav77** [comment]
Sir please reveiw my PR

**Comment by tomatillos** [comment]
Hi, could you resolve the merge conflicts, and remove any edits to files that are not relevant to the PR (e.g. the xml files etc)

**Comment by utkarsh-shrivastav77** [comment]
I revert all the files but still there is conflicts please check it

**Comment by utkarsh-shrivastav77** [comment]
Sir what's the error now

**Pull request closed without merging**